tests/ModelContextProtocol.Tests/Configuration/McpServerBuilderExtensionsToolsTests.cs

```diff
@@ -20,13 +20,20 @@ namespace ModelContextProtocol.Tests.Configuration;
 
 public partial class McpServerBuilderExtensionsToolsTests : ClientServerTestBase
 {
+    private MockLoggerProvider _mockLoggerProvider = new();
```

Comment:
This test could be updated to use the MockLoggerProvider in the base class if #1054 gets merged first.